libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/RouteRefreshExtra.kt

```diff
@@ -0,0 +1,42 @@
+package com.mapbox.navigation.core.routerefresh
+
+import androidx.annotation.StringDef
+import com.mapbox.api.directions.v5.models.Incident
+import com.mapbox.api.directions.v5.models.LegAnnotation
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+
+/**
+ * Extra data of route refresh
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+object RouteRefreshExtra {
+
+    /**
+     * The state becomes [REFRESH_STATE_STARTED] when route refresh round is started
+     */
+    const val REFRESH_STATE_STARTED = "STARTED"
+
+    /**
+     * The state becomes [REFRESH_STATE_FINISHED_SUCCESS] when route refresh successfully refresh
+     */
+    const val REFRESH_STATE_FINISHED_SUCCESS = "FINISHED_SUCCESS"
+
+    /**
+     * The state becomes [REFRESH_STATE_FINISHED_FAILED] when route refresh is failed and a route
```

Comment:
"route is going to be cleaned up"
BTW, I'm not sure about "gonna". It's kinda informal, go we use such language in docs?

Reply:
let's replace with `is cleaned up`:
`The state becomes [REFRESH_STATE_FINISHED_FAILED] when route refresh is failed and the route is cleaned up of invalid data (if it's not yet):